src/translations/translations.qrc

```diff
@@ -14,5 +14,6 @@
         <file alias="sqlb_ko_KR">sqlb_ko_KR.qm</file>
         <file alias="sqlb_tr">sqlb_tr.qm</file>
         <file alias="sqlb_uk_UA">sqlb_uk_UA.qm</file>
+	<file alias="sqlb_it">sqlb_it.qm</file>
```

Comment:
Seems to be a mismatch with spaces vs tabs here.

In this instance don't worry about fixing it.  I'll merge these commits now and manually tweak it right after as I'm just about to build a few test nightly builds anyway. :smile:

Reply:
Yes, I used tab, probably there were spaces but the editor doesn't show the difference.